terraform-provider-constellation/internal/provider/provider_test.go

```diff
@@ -23,12 +23,14 @@ const (
 	testingConfig = `provider "constellation" {}`
 )
 
+var version = "v2.13.0" // inject existing release to test if default version behavior is correct (lockstep)
```

Comment:
This should be injected dynamically and set to some error-causing version otherwise (e.g. empty), so that we don't accidentally expect injection and have a working version instead.